### ISSUE TITLE
docs in md

### ISSUE THREAD

**Issue by edenvidal** [weareunderdesign/rnbw#881]
- [ ] @stanislavasal since we're continually editing the colorpaper at notion, let's use `https://cdnjs.cloudflare.com/ajax/libs/marked/2.0.3/marked.min.js` and connect it to a file like `colorpaper.md` - so that we can then replace the file directly anytime we make a change. (nothing changes for the users)

**Comment by edenvidal** [comment]
@stanislavasal gonna do the same thing for the handy documentation. it'll tame a moment but then saves a lot of time. lmk your thoughts/questions.

**Comment by stanislavasal** [comment]
@edenvidal wow, i didn't know about such things! i will start with it tomorrow morning :)

**Comment by stanislavasal** [comment]
@edenvidal (great idea)

**Comment by stanislavasal** [comment]
@edenvidal hello! i have created and connected the files. two things: 
1. the table from notion is automatically exported with this link `[tools](rnbw%2073d1033d6e0b471495349771c1545e12/tools%20e5a2be8622eb44148834e0c6b4eef89b.csv) `and because of it the table does not render. what do you think, should we edit it manually, or leave it as it is? 
2. i added these files to rnbw.design since colorpaper is there, and i ran them through the local server until we publish them. should i move them to rnbw? thank you

**Comment by stanislavasal** [comment]
https://github.com/user-attachments/assets/c80e7c66-203b-4290-b321-6ca957a0ea5c

**Comment by edenvidal** [comment]
got it @stanislavasal i think it is ok if it won't show the table. no worries.

so, now, if i update the file in notion and let you know - it is just a matter of updating it. right?
and, where is the table of contents? why is it gone?

**Comment by stanislavasal** [comment]
@edenvidal yes, now i should just reexport it and put the new one. not sure i got you about table, as i wrote, notion does not export it right, so that is why our code does not render it right, but we can fix it manually. but as i understood "i think it is ok if it won't show the table. no worries" we don't need to do it, right?

**Comment by edenvidal** [comment]
yes @stanislavasal . the table is not as important to manually adjust it. the idea is to replace files quickly when there's a change.

**Comment by stanislavasal** [comment]
@edenvidal nice :) will push it than and make the same for handy

**Comment by stanislavasal** [comment]
@edenvidal done, can i close it?

**Comment by edenvidal** [comment]
please do!
